SEED-platform/seed
# ULID, UBID, and Footprints

#### Any background context you want to provide?
SEED can now handle more advanced geometry-based operations now that the database is PostGIS-based.

#### What's this PR do?
* Add ULID as a matching field
* Parse building footprints in WKT format 
* Add centroid, lat, and long to taxlot
* Add popup of building footprint

#### How should this be manually tested?
* import san jose data
* map Property Coordinates to Property Footprint
* verify that the property footprint popup works
* verify that the ulid, ubid are parsed correctly by verifying the existence of lat/long in the data
* test exporting the GeoJSON and importing into geojson.io.

#### What are the relevant tickets?
#1855 #1856 

#### Screenshots (if appropriate)
